components/actionbutton/metadata/actionbutton.yml

```diff
@@ -652,3 +652,500 @@ examples:
           </button>
         </div>
       </div>
+
+  - id: actionbutton
```

Comment:
Could you make each of the `id`s in this file unique?

Ideally, I think we'd have some kind of linter running on these to prevent duplicative `id`s, but that's for future work.

Reply:
I thought it had implications for VRT! Maybe I've been misunderstanding its importance :-)

Reply:
Hmmm I don't think it does, but I may be wrong... I know VRT isn't smart enough to realize when the examples get out of order, so it's clearly not using an id, but you know what? *IT SHOULD!*

You're a genius, @pfulton